pkg/cmd/cli/cmd/logs.go

```diff
@@ -63,6 +68,7 @@ func NewCmdLogs(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Com
 		}
 		cmdutil.CheckErr(o.RunLog())
 	}
+	cmd.Flags().Int("version", -1, "View the logs of the deployment or build associated with the provided version")
```

Comment:
Should default to zero.  `View the logs of a particular build or deployment by version if greater than zero`.